rocker-gcc-san/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rocker:r-devel-san
+
+RUN useradd docker \
+	&& mkdir /home/docker \
+	&& chown docker:docker /home/docker \
+	&& addgroup docker staff
+
+ENV RHUB_PLATFORM linux-x86_64-rocker-gcc-san
```

Comment:
This should refer to sg from here: https://github.com/r-hub/sysreqsdb/tree/master/platforms

I fixed this, too.